applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -340,14 +363,47 @@ fn handle_connection(
         .map_err(|e| MmProxyError::OtherError(format!("Error reading from stream {}", e)))?;
     let date = Local::now();
     let request_string = stringify_request(&buffer[..]);
+    debug!(target: LOG_TARGET, "Request: {}", request_string);
     let request_type = get_request_type(&buffer[..])?;
     let url_part = get_url_part(&buffer[..])?;
     if request_type.starts_with("GET") {
-        debug!(target: LOG_TARGET, "Handling Method: {}", url_part);
+        info!(target: LOG_TARGET, "Handling GET Method: {}", url_part);
         // GET requests
         let url = format!("{}{}", MONEROD_URL, url_part);
         let mut curl = base_curl(0, &url, false)?;
         let data = do_curl(&mut curl, b"")?;
+        match url_part.as_str() {
+            "/getheight" => match grpcclientresult {
+                Ok(mut grpcclient) => {
+                    match rt.block_on(grpcclient.get_tip_info(grpc::Empty {})) {
+                        Ok(result) => {
+                            let height = result.into_inner().metadata.unwrap().height_of_longest_chain;
+                            // Short Circuit XMRig to request a new block template
+                            // TODO: needs additional testing
+                            if let Some(current_height) = transient.tari_height {
+                                if height != current_height {
+                                    adjust_height(&data, current_height)?;
+                                    info!(target: LOG_TARGET, "Tari tip changed");
+                                } else if let Some(submit_height) = transient.tari_prev_submit_height {
+                                    if submit_height >= current_height {
+                                        adjust_height(&data, current_height)?;
+                                        info!(target: LOG_TARGET, "Already submitted for current Tari height");
```

Comment:
```suggestion
                                        debug!(target: LOG_TARGET, "Already submitted for current Tari height");
```
Non technical user I dont think wants to see this